src/uu/head/src/constants.rs

```diff
@@ -0,0 +1,89 @@
+
+pub const EXIT_FAILURE: i32 = 1;
```

Comment:
I value a lot consistency as we have a lot of binaries.
Please move that into head.rs and follow what is done elsewhere.
example:
https://github.com/uutils/coreutils/blob/master/src/uu/ls/src/ls.rs#L43-L52
and
https://github.com/uutils/coreutils/blob/master/src/uu/ls/src/ls.rs#L78-L82

or
https://github.com/uutils/coreutils/blob/master/src/uu/chroot/src/chroot.rs#L21-L40

thanks and sorry for the extra work

Reply:
Do I understand you correctly that I should move the content of `app.rs` to `head.rs`?

Reply:
Move constants to `app.rs` and instead of functions use an options mod. 

Instead of this:

```rust
#[inline(always)]
pub fn bytes_name() -> &'static str {
    "BYTES"
}

#[inline(always)]
pub fn lines_name() -> &'static str {
    "LINES"
}
```


Use this:

```rust
pub mod Options {
    pub static LINES: &str = "lines";
    pub static BYTES: &str = "bytes";
}
```

Instead of this:

```rust
#[inline(always)]
pub fn usage() -> &'static str {
    "head [FLAG]... [FILE]..."
}
```

Use this:

```rust
static USAGE: &str = "head [FLAG]... [FILE]...";
```

etc.